src/destination.js

```diff
@@ -0,0 +1,16 @@
+class Destination {
+  constructor(destination) {
+    this.id = destination.id;
+    this.destination = destination.destination;
+    this.estimatedLodgingCostPerDay = destination.estimatedLodgingCostPerDay;
+    this.estimatedFlightCostPerPerson = destination.estimatedFlightCostPerPerson;
+    this.image = destination.image;
+    this.alt = destination.alt;
+  }
+
+  calculateEstimatedCost() {
+
+  }
+}
+
```

Comment:
I went back and forth about adding a Destination class and ended up not doing so, but I also don't think having one is a bas idea. I decided to have the calculate cost method live on the trip instead. I'll be curious to see how useful it is for you!